rpc/handler.go

```diff
@@ -435,6 +440,46 @@ func waitForUserOpCompletion(ctx context.Context, ethClient *ethclient.Client,
 	}
 }
 
+func handleEthEstimateGas(ctx context.Context, c *gin.Context,
```

Comment:
@adelowo have you seen [github.com/stackup-wallet/stackup-bundler/main/pkg/gas/estimate.go#L82](https://github.com/stackup-wallet/stackup-bundler/blob/f145dfd14d12956176b48b80c50d6cb44a93ca49/pkg/gas/estimate.go#L82)?
It reports on all the 4337 gas metrics.